api/users/serializers.py

```diff
@@ -1,6 +1,37 @@
 from rest_framework import serializers as ser
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 from website.models import User
+from rest_framework.fields import SkipField, get_attribute
+
+
+class CharFieldWithReadDefault(ser.CharField):
```

Comment:
These new fields are specific to users; therefore, they belong in `api.base.serializers`.